### DISCUSSION TITLE
Searching in hidden files

### DISCUSSION THREAD

**Discussion by Jomik** [LazyVim/LazyVim#2613] [Q&A]
Is there a good way for us to use the `<space>f...` shortcuts to include hidden files?
I frequently need to edit workflows in `.github/`.

**Comment by dpetka2001** [comment]
If you have the Telescope picker open, you can press `<a-h>` to extend the search to also show hidden files. See [here](https://github.com/LazyVim/LazyVim/blob/91126b9896bebcea9a21bce43be4e613e7607164/lua/lazyvim/plugins/editor.lua#L238). Also, `<c-/>` in Telescope prompt insert mode or `?` in prompt normal mode, should show the available keymaps for the picker. Or make a new keymap for the Telescope picker that you want passing it the option `hidden = true`.
For example `require("telescope.builtin").find_files({hidden=true, layout_config={prompt_position="top"}})`
